test/acs-engine-test/promoteToFailure/promoteToFailure.go

```diff
@@ -0,0 +1,219 @@
+package promoteToFailure
```

Comment:
We should change this to adhere to golang idioms (specifically, `anUpperCaseName` is not recommended). See:

https://blog.golang.org/package-names

How about `ptf`?

Reply:
im trying to keep this as close to the work shruti did as possible just so its easy to compare notes when things arent working.

Reply:
alright just changed this to be promote